### ISSUE TITLE
Status da oportunidade

### ISSUE THREAD

**Issue by wellfreire** [iMastersDev/oportunidades#22]
Talvez para um segundo milestone, seria interessante a opção de modificar o status de uma oportunidade de 'em aberto' para 'vaga preenchida', ou até mesmo a exclusão da mesma pelo autor assim que atendida.

Isto ajuda a evitar que tanto o anunciante de uma oportunidade recente, mas já preenchida, continue recebendo e-mails sobre a mesma, quanto o candidato pra que não perca tempo.

**Comment by netojoaobatista** [comment]
+1 para a feature "status da oportunidade"

**Comment by samuelmesq** [comment]
+1, feature essencial

**Comment by wellfreire** [comment]
Alguém me ensina como colocar a label 'feature' na issue. Novo no Github galera, valeu!

**Comment by augustohp** [comment]
@wellfreire Do lado direito da _issue_ tem uma engrenagem ao lado de **Labels**, clicando na engrenagem você consegue especificar as _labels_ que você quer.

+1 pra essa feature. Podemos também determinar um tempo em que a vaga é válida, algo em terno de 15-30 dias. Depois disos a vaga é fechada automaticamente.

Nada que um cron não resolva... =P

Digo isso, porque precisamos trabalhar em formas mais simples e ir iterando nas funcionalidades. Não sabemos ainda qual é boa ou não, saberemos na prática né =D

**Comment by iannsp** [comment]
essa feature realmente eh fundamental, eu digo que deve estar nesse primeiro sprint.

**Comment by wellfreire** [comment]
@augustohp valeu pela dica! 

Quando você diz que 'podemos determinar' um tempo em que a vaga é válida, quer dizer que o app fará isso automaticamente? ou o autor da oportunidade é quem escolhe em quanto tempo expira?

**Comment by iannsp** [comment]
@augustohp @wellfreire a ferramenta tem um valor default, alteravel. esse seria o melhor caminho, creio eu.

**Comment by wellfreire** [comment]
concordo com você @iannsp, o ideal é que tenha um default com a liberdade para que o autor escolha o prazo.

**Comment by augustohp** [comment]
@wellfreire Isso mesmo!

@iannsp Só acho que essa estória não cabe nos primeiros sprints. Precisamos primeiro focar em ter um cadastro de oportunidade e depois em exibir. Essa história só pode vir depois disso.

**Comment by iannsp** [comment]
houve um problema de comunicacao aqui.
Eu interpretei todo o hangout como um sprint e por isso a frase acima.
comunicacao resolvida.

**Comment by augustohp** [comment]
Então essa _issue_ vai pro backlog e decidimos na hora quando é a hora de colocar ela dentro de algum _sprint_.

@wellfreire Mandou bem pra caralho na sugestão!

**Comment by netojoaobatista** [comment]
Mandou muito bem mesmo @wellfreire. Esse é um recurso fundamental.

**Comment by wellfreire** [comment]
Essa sugestão foi por experiência própria, pois sempre fico na dúvida quando vejo uma vaga de alguns dias e não sei se já foi preenchida.
